xrpl/ledger-entry-types/amm.go

```diff
@@ -48,33 +61,51 @@ func (a *AMM) UnmarshalJSON(data []byte) error {
 	return nil
 }
 
-type AMMAsset struct {
-	Currency string
-	Issuer   types.Address
+// ---------------------------------------------
+// Asset Object
+// ---------------------------------------------
+
+type Asset struct {
```

Comment:
Where did you found the fields of the `Asset`? 🤔 I cannot find them in the docs

Reply:
I looked at this page in the docs:
https://xrpl.org/docs/references/protocol/ledger-data/ledger-entry-types/amm#amm-fields

<img width="861" alt="image" src="https://github.com/user-attachments/assets/91b2a1f9-7ce6-491f-a0bc-8b9a0391a3d8">


And also I looked at xrpl.js